File: ToolPageAppClientScripts/Model/Formatter.cs
using Bridge;
using System;
using System.Linq;
using UI5;
using System.Collections.Generic;

namespace ClientScripts
{
    [FileName("model/formatter.js")]
    [Name("manu.toolpageapp.fv.model.Formatter")]
    public class Formatter
    {
        /// <summary>
        /// Gets the path to the image based on the device
        /// </summary>
        /// <param name="bIsPhone">the value to be checked</param>
        /// <returns>path to image</returns>
        public virtual string srcImageValue(bool bIsPhone)
        {
            var sImageSrc = "";
            if (!bIsPhone) {
                sImageSrc = "/images/homeImage.jpg";
            } else {
                sImageSrc = "/images/homeImage_small.jpg";
            }
            return sImageSrc;
        }

        [Init(InitPosition.Bottom)]
        public static void DefineScript()
        {
            sap.ui.define(new string[] {
                },
                new Func<object>(
                    () => {
                        Formatter newObj = Glue.CreateRawClassObject<Formatter>();
                        return newObj;
                    }
                )
            );
        }
    }
}

File: ToolPageApp/wwwroot/ui5/model/formatter.js
Bridge.assembly("ClientScripts", function ($asm, globals) {
    "use strict";

    Bridge.define("manu.toolpageapp.fv.model.Formatter", {
        $metadata : function () { return {"att":1048577,"a":2,"m":[{"a":2,"isSynthetic":true,"n":".ctor","t":1,"sn":"ctor"},{"a":2,"n":"DefineScript","is":true,"t":8,"sn":"DefineScript","rt":System.Void},{"v":true,"a":2,"n":"srcImageValue","t":8,"pi":[{"n":"bIsPhone","pt":System.Boolean,"ps":0}],"sn":"srcImageValue","rt":System.String,"p":[System.Boolean]}]}; },
        methods: {
            /**
             * Gets the path to the image based on the device
             *
             * @instance
             * @public
             * @this manu.toolpageapp.fv.model.Formatter
             * @memberof manu.toolpageapp.fv.model.Formatter
             * @param   {boolean}    bIsPhone    the value to be checked
             * @return  {string}                 path to image
             */
            srcImageValue: function (bIsPhone) {
                var sImageSrc = "";
                if (!bIsPhone) {
                    sImageSrc = "/images/homeImage.jpg";
                } else {
                    sImageSrc = "/images/homeImage_small.jpg";
                }
                return sImageSrc;
            }
        }
    });
});

sap.ui.define(System.Array.init([], System.String), function () {
    var newObj = UI5.Glue.CreateRawClassObject(manu.toolpageapp.fv.model.Formatter);
    return newObj;
});
